vars/capsuleSystemTests.groovy

```diff
@@ -28,6 +28,7 @@ void call(Map additionalConfig=[:], parametersOverride=[:]) {
     options {
       ansiColor('xterm')
       timestamps()
+      timeout(time: 3, unit: 'HOURS')
```

Comment:
We cannot do that here. because some of the runs are designed to run for a few days